packages/terra-search-field/src/SearchField.jsx

```diff
@@ -273,17 +272,17 @@ class SearchField extends React.Component {
     return (
       <div {...customProps} className={searchFieldClassNames}>
         <div className={cx('input-group')}>
-          <Input
+          <input
             className={cx('input')}
             type="search"
             placeholder={placeholder}
             onChange={this.handleTextChange}
             disabled={isDisabled}
-            ariaLabel={inputText}
+            aria-label={inputText}
             aria-disabled={isDisabled}
             onKeyDown={this.handleKeyDown}
             onInput={this.handleInput}
-            refCallback={this.setInputRef}
+            ref={this.setInputRef}
             {...additionalInputAttributes}
```

Comment:
Previously, form-input will automatically add `aria-required` if `required` was passed to `inputAttributes`. Since this is no longer leveraging form-input, it would lose this logic. Should [this](https://github.com/cerner/terra-core/blob/cb664693fa409abe01ad4118a95877015afbdabf/packages/terra-form-input/src/Input.jsx#L147-L149) be added here?

Reply:
I and @neilpfeiffer discussed this and decided that it is unneeded to make `search-field` a required field.